2025-09-17 01:00:03.384685+09:00 cron_1hour start
2025-09-17 01:00:04.453121+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  27.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   2.0  0.1  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...  25.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...  18.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...  14.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...   0.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...   7.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...   0.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...  20.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 02:00:02.576708+09:00 cron_1hour start
2025-09-17 02:00:03.752621+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  34.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   7.0  0.1  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...   0.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   5.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   0.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...  20.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...  14.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  25.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...   0.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...  20.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 03:00:02.836397+09:00 cron_1hour start
2025-09-17 03:00:03.115952+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  29.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...  29.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...  20.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   0.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...  16.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...  29.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  29.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...  25.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...  16.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 04:00:02.202185+09:00 cron_1hour start
2025-09-17 04:00:02.603354+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  23.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   0.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...   0.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   0.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...  18.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...  23.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  32.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...  27.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...   0.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 05:00:02.613983+09:00 cron_1hour start
2025-09-17 05:00:03.190071+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  25.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   0.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...   0.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   0.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...  14.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...   0.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  25.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...  27.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...  32.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 06:00:02.167672+09:00 cron_1hour start
2025-09-17 06:00:04.584449+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  34.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   0.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...   0.0  0.1  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   5.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   2.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...   0.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...   0.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  29.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...  36.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...   0.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 07:00:03.432082+09:00 cron_1hour start
2025-09-17 07:00:03.962904+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  23.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...  25.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...  32.0  0.1  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   9.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
130  288  128.7441  35.4915  ...  11.0  0.0  POINT (128.74410 35.49150)
131  289  127.8791  35.4130  ...   0.0  0.0  POINT (127.87910 35.41300)
132  294  128.6046  34.8882  ...  27.0  0.0  POINT (128.60460 34.88820)
133  295  127.9264  34.8166  ...  11.0  0.0  POINT (127.92640 34.81660)
134  296  128.9602  35.2178  ...   0.0  0.0  POINT (128.96020 35.21780)

[135 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 08:00:02.988376+09:00 cron_1hour start
2025-09-17 08:00:03.233031+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...  25.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   0.0  0.1  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...  25.0  1.1  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   0.0  0.1  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
130  288  128.7441  35.4915  ...  32.0  0.0  POINT (128.74410 35.49150)
131  289  127.8791  35.4130  ...   0.0  0.0  POINT (127.87910 35.41300)
132  294  128.6046  34.8882  ...  25.0  0.0  POINT (128.60460 34.88820)
133  295  127.9264  34.8166  ...   0.0  0.0  POINT (127.92640 34.81660)
134  296  128.9602  35.2178  ...   0.0  0.0  POINT (128.96020 35.21780)

[135 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 09:00:02.284479+09:00 cron_1hour start
2025-09-17 09:00:02.535272+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD    RN                    geometry
0     90  128.5647  38.2509  ...  29.0   0.2  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...   0.0   0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...   0.0  21.8  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   0.0   0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   9.0   3.9  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...   ...                         ...
130  288  128.7441  35.4915  ...  11.0   0.0  POINT (128.74410 35.49150)
131  289  127.8791  35.4130  ...  27.0   0.0  POINT (127.87910 35.41300)
132  294  128.6046  34.8882  ...  25.0   0.0  POINT (128.60460 34.88820)
133  295  127.9264  34.8166  ...  27.0   0.0  POINT (127.92640 34.81660)
134  296  128.9602  35.2178  ...  20.0   0.0  POINT (128.96020 35.21780)

[135 rows x 11 columns]
error (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'host.docker.internal' ([Errno 111] Connection refused)")
(Background on this error at: https://sqlalche.me/e/20/e3q8)
OrderedDict([('DB_ID', 'root'), ('DB_PW', 'test'), ('DB_NAME', 'golf'), ('PORT', '13309'), ('SERVICE_KEYS', '3b8dcb53f1ccdd05fb434481223a53ff8fe1a47df3860abbb1a315ddf2637338,'), ('b4334a15c2d153a244e846cf40edf1532f9bf56f262a9cc4ea5e2522ea38f9ea,', None), ('420b63b417a99ff90d24fb6538f51d9a3785c2e2756a544c05d27c9ebf0c0673,', None), ('89d11f51ba180d310e2afc89e6e9b9ce2b190d2bbac6eb6dedf116ebd641b6d2,', None), ('0a0633bc3348a83dc93f4b0516f2d5877db153b07792a880a3645c677029ce44', None), ('KMA_KEYS', 'K66mUw9zTByuplMPczwchQ'), ('FRONT_PORT', '3000'), ('BACKEND_FORT', '8000')])
    STN                   TM    TA   PR    HM   WS    WD   RN
0    90  2025-09-18 11:00:00  21.7  0.0  59.0  2.4   5.0  0.0
1    93  2025-09-18 11:00:00  21.5  0.0  63.0  1.1  34.0  0.0
2    95  2025-09-18 11:00:00  22.7  0.0  63.0  2.7  27.0  0.0
3    98  2025-09-18 11:00:00  22.8  0.0  77.0  1.0   9.0  0.0
4    99  2025-09-18 11:00:00  22.7  0.0  83.0  2.6   9.0  0.0
..  ...                  ...   ...  ...   ...  ...   ...  ...
92  288  2025-09-18 11:00:00  24.4  0.0  73.0  2.7   5.0  0.0
93  289  2025-09-18 11:00:00  24.7  0.0  79.0  1.6   7.0  0.0
94  294  2025-09-18 11:00:00  24.4  0.0  73.0  2.0  11.0  0.0
95  295  2025-09-18 11:00:00  25.1  0.0  78.0  1.7  36.0  0.0
96  296  2025-09-18 11:00:00  23.6  0.0  79.0  3.5   5.0  0.0

[97 rows x 8 columns]
2025-09-18 12:00:03.559412+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...   5.0  0.0  POINT (128.56470 38.25090)
1     93  127.7544  37.9474  ...  34.0  0.0  POINT (127.75440 37.94740)
2     95  127.3042  38.1479  ...  27.0  0.0  POINT (127.30420 38.14790)
3     98  127.0607  37.9019  ...   9.0  0.0  POINT (127.06070 37.90190)
4     99  126.7665  37.8859  ...   9.0  0.0  POINT (126.76650 37.88590)
..   ...       ...      ...  ...   ...  ...                         ...
131  288  128.7441  35.4915  ...   5.0  0.0  POINT (128.74410 35.49150)
132  289  127.8791  35.4130  ...   7.0  0.0  POINT (127.87910 35.41300)
133  294  128.6046  34.8882  ...  11.0  0.0  POINT (128.60460 34.88820)
134  295  127.9264  34.8166  ...  36.0  0.0  POINT (127.92640 34.81660)
135  296  128.9602  35.2178  ...   5.0  0.0  POINT (128.96020 35.21780)

[136 rows x 11 columns]
OrderedDict([('DB_ID', 'root'), ('DB_PW', 'test'), ('DB_NAME', 'golf'), ('PORT', '13309'), ('SERVICE_KEYS', '3b8dcb53f1ccdd05fb434481223a53ff8fe1a47df3860abbb1a315ddf2637338,'), ('b4334a15c2d153a244e846cf40edf1532f9bf56f262a9cc4ea5e2522ea38f9ea,', None), ('420b63b417a99ff90d24fb6538f51d9a3785c2e2756a544c05d27c9ebf0c0673,', None), ('89d11f51ba180d310e2afc89e6e9b9ce2b190d2bbac6eb6dedf116ebd641b6d2,', None), ('0a0633bc3348a83dc93f4b0516f2d5877db153b07792a880a3645c677029ce44', None), ('KMA_KEYS', 'K66mUw9zTByuplMPczwchQ'), ('FRONT_PORT', '3000'), ('BACKEND_FORT', '8000')])
     STN                   TM    TA   PR    HM   WS    WD   RN
0     90  2025-09-18 11:00:00  21.7  0.0  59.0  2.4   5.0  0.0
1     93  2025-09-18 11:00:00  21.5  0.0  63.0  1.1  34.0  0.0
2     95  2025-09-18 11:00:00  22.7  0.0  63.0  2.7  27.0  0.0
3     98  2025-09-18 11:00:00  22.8  0.0  77.0  1.0   9.0  0.0
4     99  2025-09-18 11:00:00  22.7  0.0  83.0  2.6   9.0  0.0
..   ...                  ...   ...  ...   ...  ...   ...  ...
189  288  2025-09-18 12:00:00  24.3  0.8  72.0  2.1   5.0  0.0
190  289  2025-09-18 12:00:00  25.1 -0.1  73.0  0.8  11.0  0.0
191  294  2025-09-18 12:00:00  24.8  0.5  70.0  1.8  11.0  0.0
192  295  2025-09-18 12:00:00  25.6 -0.1  76.0  1.5  36.0  0.0
193  296  2025-09-18 12:00:00  23.3  0.5  79.0  5.4   9.0  0.0

[194 rows x 8 columns]
2025-09-18 12:04:02.993387+09:00 최종 현재부터 6시간 이후 판다스 호출 완료 :      STN       LON      LAT  ...    WD   RN                    geometry
0     90  128.5647  38.2509  ...   5.0  0.0  POINT (128.56470 38.25090)
1     90  128.5647  38.2509  ...   2.0  0.0  POINT (128.56470 38.25090)
2     93  127.7544  37.9474  ...  34.0  0.0  POINT (127.75440 37.94740)
3     93  127.7544  37.9474  ...   5.0  0.0  POINT (127.75440 37.94740)
4     95  127.3042  38.1479  ...  27.0  0.0  POINT (127.30420 38.14790)
..   ...       ...      ...  ...   ...  ...                         ...
267  294  128.6046  34.8882  ...  11.0  0.0  POINT (128.60460 34.88820)
268  295  127.9264  34.8166  ...  36.0  0.0  POINT (127.92640 34.81660)
269  295  127.9264  34.8166  ...  36.0  0.0  POINT (127.92640 34.81660)
270  296  128.9602  35.2178  ...   5.0  0.0  POINT (128.96020 35.21780)
271  296  128.9602  35.2178  ...   9.0  0.0  POINT (128.96020 35.21780)

[272 rows x 11 columns]
